client/log_client_test.go

```diff
@@ -36,13 +37,15 @@ func TestAddGetLeaf(t *testing.T) {
 // addSequencedLeaves is a temporary stand-in function for tests until the real API gets built.
 func addSequencedLeaves(ctx context.Context, env *integration.LogEnv, client *LogClient, leaves [][]byte) error {
 	// TODO(pavelkalinnikov): Replace with AddSequencedLeaves batch API.
-	for _, l := range leaves {
-		if err := client.QueueLeaf(ctx, l); err != nil {
-			return err
+	for i, l := range leaves {
+		if err := client.AddSequencedLeaf(ctx, l, int64(i)); err != nil {
+			return fmt.Errorf("AddSequencedLeaf(): %v", err)
 		}
-		env.Sequencer.OperationSingle(ctx)
+	}
+	env.Sequencer.OperationSingle(ctx)
+	for _, l := range leaves {
```

Comment:
Would just waiting for N-th entry inclusion work? Looks like it implies other entries are there as well.

Reply:
Done